protocol/x/clob/memclob/memclob.go

```diff
@@ -1698,8 +1698,12 @@ func (m *MemClobPriceTimePriority) mustPerformTakerOrderMatching(
 
 		// affiliate revshare doesnt affect subaccount balance, we can pass an empty map
```

Comment:
Let's use below comment:

```
Pass in empty map to avoid reading `AffiliateWhitelist` from state in every `CheckTx`. This deviates from `DeliverTx` which accounts for affiliate whitelist correctly. 
This deviation is ok becasue rev shares/fees are distributed to the recipient’s bank balance and not settled at the subaccount level, and won’t affect the collateralization of future operations in the operations queue.
```
```